src/main/java/io/cdap/plugin/gcp/bigquery/sink/AbstractBigQuerySinkConfig.java

```diff
@@ -69,6 +87,21 @@ public boolean isAllowSchemaRelaxation() {
     return allowSchemaRelaxation;
   }
 
+  @Nullable
+  public Boolean isCreatePartitionedTable() {
+    return createPartitionedTable;
+  }
+
+  @Nullable
+  public String getPartitionByField() {
+    return partitionByField;
+  }
+
+  @Nullable
+  public Boolean isPartitionFilterRequired() {
```

Comment:
similar comment, would be simpler if this returns a boolean, returning false if it's null.

Reply:
Done.